webapi/middleware.go

```diff
@@ -162,28 +163,68 @@ func broadcastTicket() gin.HandlerFunc {
 		}
 
 		dcrdClient := c.MustGet("DcrdClient").(*rpc.DcrdRPC)
+		var e *wsrpc.Error
+
+		if request.ParentHex != nil {
+			parentTx, err := decodeTransaction(*request.ParentHex)
+			if err != nil {
+				log.Errorf("%s: Failed to decode parent hex (ticketHash=%s): %v", funcName, request.TicketHash, err)
```

Comment:
Hmm I guess we don't actually need to fail and return an error here. There is a chance the provided ticket tx could be valid even if the parent tx is not.

We could still attempt to validate/broadcast the ticket tx, and if that fails, return an error at that point.

```
// If this errors, just log an error and continue. There is a chance the provided ticket tx could be valid even if the parent tx is not.
log.Warnf("%s: Failed to decode parent hex (ticketHash=%s): %v", funcName, request.TicketHash, err)

Reply:
I think if it is supplied it should be valid to prevent using the vsp to send any tx